### ISSUE TITLE
Support GA4

### ISSUE THREAD

**Issue by mjpitz** [mjpitz/go-ga#1]
Analytics starting to force folks over to using gtag instead of analytics js. There are some slight differences in how these operate.

https://developers.google.com/analytics/devguides/collection/upgrade/analyticsjs

**Comment by mjpitz** [comment]
This looks like a promising resource: https://developers.google.com/analytics/devguides/collection/gtagjs

**Comment by sneko** [comment]
@mjpitz just looking around if there is library to do so.

Did you work on it since then? Otherwise I found GA4 compatible ones:
https://github.com/ystyle/google-analytics
https://github.com/silinternational/ga-event-tracker

(not that easy to found them, it seems no one has migrated from UA 😄 )

**Comment by mjpitz** [comment]
@sneko yeah, I wasn't looking for a GA4 compatible one at the time... but I also remember seeing some note about Google deprecating the new one... which isn't surprising.
